events/test_routes.py

```diff
@@ -1,80 +1,89 @@
 """Unit tests for events service HTTP routes."""
 
 import unittest
-from unittest.mock import patch
 import datetime
 from bson import json_util
 import mongomock
 from app import app
-from eventclass import Event
 
 EXAMPLE_TIME = datetime.datetime(2019, 6, 11, 10, 33, 1, 100000)
 
-VALID_INFO = {
+VALID_REQUEST_INFO = {
+    'event_name': 'valid_event',
+    'description': 'This event is formatted correctly!',
+    'author_id': 'admin',
+    'event_time': EXAMPLE_TIME}
+INVALID_REQUEST_INFO_MISSING_ATTRIBUTE = {
+    'event_name': 'invalid_event_missing',
+    'description': 'This event is missing an author!',
+    'event_time': EXAMPLE_TIME}
+
+VALID_DB_EVENT = {
     'name': 'valid_event',
     'description': 'This event is formatted correctly!',
     'author': 'admin',
-    'event_time': EXAMPLE_TIME}
-VALID_INFO_WITH_ID = {
+    'event_time': EXAMPLE_TIME,
+    'created_at': EXAMPLE_TIME,
+    'event_id': 'unique_event_id0'}
```

Comment:
Did we want to manually create event IDs or use the auto-generated IDs from MongoDB?

Reply:
I think they should be defined by the DB but I'd have to change my Event class for that - will probably do it in a later PR

Reply:
Sounds good.